src/Detail.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react'
+import { GetPoke } from './api'
+import PokeItem from './PokeItem';
+
+export default class Detail extends Component {
+    state = { character: {} }
+
+    async componentDidMount() {
+        const data = await GetPoke(this.props.match.param.pokeId);
```

Comment:
this is the main reason your detail card isnt showing, its because you mispelled params as param